### ISSUE TITLE
how to use RAGAS metrics for GPT3.5 Finetuned model

### ISSUE THREAD

**Issue by s-kumar17** [explodinggradients/ragas#135]
Hi all,
I understand how RAGAS work for RAG system.
I have use case where I have fine-tuned a GPT3.5 model on my data and using this model for question-answering.
I want to know how if I can use RAGAS to evaluate this fine-tuned model, as it does not have context/chunks to be passed into RAGAS metrics function.
Can anyone help me with "how to use RAGAS metrics for my GPT3.5 Finetuned model"

**Comment by jjmachan** [comment]
Hey @s-kumar17 thanks for raising the question. You could use the `faithfulness` and `answer_relevancy` metric here. Please refer to this to see it in action https://gpt-index.readthedocs.io/en/stable/examples/finetuning/openai_fine_tuning.html#evaluation
